lib/blockscore/person.rb

```diff
@@ -10,7 +10,7 @@ class Person < Base
 
     attr_reader :question_sets
 
-    def initialize(options = {})
+    def initialize(*)
```

Comment:
While we are on the topic of super, some people subscribe to calling super last, some first, and some call it when appropriate in the constructor. Which camp are you in?